ios_app_clip/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -475,7 +498,6 @@
 			baseConfigurationReference = 9740EEB21CF90195004384FC /* Debug.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
```

Comment:
This was causing an error because the asset catalog doesn't define `AccentColor`.